src/main/webapp/app/overview/course-discussion/course-discussion.component.ts

```diff
@@ -245,4 +248,14 @@ export class CourseDiscussionComponent extends CourseDiscussionDirective impleme
             this.onSelectPage();
         }
     }
+
+    /**
+     * fetches next page of posts if currently rendered posts are approaching the end of the available list of elements
+     * @param event
+     */
+    onItemsRender(event: VirtualScrollRenderEvent<any>) {
+        if (!this.isLoading && event.endIndex + 1 < this.totalItems && event.endIndex + 1 > this.posts.length - 2) {
```

Comment:
can we make this more configurable with a constant? 
and please move the logic when the end of the list is approached is actually happening into the `virtual-scroll.component` (= framework component).

`virtual-scroll.component` should then invoke a delegate method or event like `reachedEndOfList()` or similar in this component (= usage of the framework component).

`course-discussion.component` can then decide what should be done (e.g. ignore the message or fetch the next page)

Reply:
Done ✔️